### DISCUSSION TITLE
Set Force download = True

### DISCUSSION THREAD

**Discussion by om2468** [duckdb/duckdb#13353] [General]
I am trying to query from a larg(ish) dataset and get the message SET force_download = true. Can I do this inside a sql statement? Many Thanks
![image](https://github.com/user-attachments/assets/3da3ca06-c436-4aeb-9ed7-bd7f1b216a95)

**Comment by Tishj** [comment]
https://duckdb.org/docs/sql/statements/set

**Reply by om2468** [reply]
thank you, I understand now
